File: JavaScriptBasics/ConditionalStatements/Exercise/LunchBreak.js
function LunchBreak(input){

let name = input[0];
let episodelength = Number(input[1]);
let breaklength = Number(input[2]);

breaklength = breaklength-(0.375*breaklength);

if(breaklength>episodelength){
    console.log("You don't have enough time to watch "+name +" , you need "+Math.ceil(episodelength-breaklength)+" more minutes.")
}else{
    console.log("You have enough time to watch "+name +" and left with "+Math.ceil(episodelength-breaklength)+" free time.")  
}
}

File: JavaBasics/ConditionalStatements/Exercise/LunchBreak.java
package EntryModule.ConditionalStatements.Exercise;

import java.util.Scanner;
public class LunchBreak {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String name = scanner.nextLine();
        int duration = Integer.parseInt(scanner.nextLine());
        int restduration = Integer.parseInt(scanner.nextLine());


        double  eat = restduration/8.0;
        double chill = restduration/4.0;

        double totaltime = duration+eat+chill;

        if(totaltime>restduration){
            System.out.printf("You don't have enough time to watch " + name + ", you need %.0f more minutes.", Math.ceil(totaltime-restduration));
        }else System.out.printf("You have enough time to watch "+ name +" and left with %.0f minutes free time.", Math.ceil(restduration-(totaltime)));

    }
}
